src/index.html

```diff
@@ -11,7 +11,181 @@
+          <label>
+          How old are you?
+            <input
+              type="number"
+              name="age"
+              min="1"
+              max="100"
+              value="12"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            Full date of birth
+            <input
+              type="date"
+              name="age"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            I accept the term of the agreement
+            <input
+              type="checkbox"
+              name="term"
+              required
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="fields-block">
+        <legend>Registration</legend>
+
+        <div class="form-field">
+          <label>
+            E-mail:
+            <input
+              type="email"
+              name="email"
+              placeholder="email@example.com"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            Password:
+            <input
+              type="password"
+              name="password"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="fields-block">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="catsLover"
+              value="yes"
+            >
+            Yes
+          </label>
+          <label>
+            <input
+              type="radio"
+              name="catsLover"
+              value="no"
+            >
+            No
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            What is your favorite color?
+            <input type="color"
+              name="color"
+              value="#000"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+            What time do you go to bed?
+            <input
+              type="time"
+              name="bedTime"
+              step="2"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          What are your favorite brand of cars?n
+          <select name="car" multiple>
+            What are your favorite brand of cars?
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </div>
+
+        <div class="form-field">
+          <label>
+            How do you rate our work?
+            <input type="range" name="rating">
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="fields-block">
+        <legend>Additional info:</legend>
+
+        <div class="form-field">
```

Comment:
[Checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) the 7th point. Use `<label>` tag here, not `div`